parts/linux/cloud-init/artifacts/cse_start.sh

```diff
@@ -23,6 +26,21 @@ JSON_STRING=$( jq -n \
 mkdir -p /var/log/azure/aks
 echo $JSON_STRING | tee /var/log/azure/aks/provision.json
 
+# arg names are defined by GA and all these are required to be correctly read by GA
+# EventPid, EventTid are required to be int. No use case for them at this point.
+EVENT_JSON=$( jq -n \
+    --arg Timestamp   "${CSE_STARTTIME}" \
+    --arg OperationId "${CSE_ENDTIME}" \
+    --arg Version     "1.23" \
```

Comment:
Didn't spot version anywhere, but if it's stored somewhere, it'd be great to replace this here and in cse_main. Or just directly in the helper func